### PR TITLE
feature/147: Updated template to use GitHub Actions instead of travis

### LINKED ISSUE #147: Update template to use Github Actions
Update the template to use GitHub Actions instead of Travis CI

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #147

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: CI Build
+
+on:
+  - push
```
Comment (AnastasiiaSergienko) [inline .github/workflows/ci-build.yml:4]: Maybe this is better?

on:
  push:
    branches:
      - main
  pull_request:

Comment (AnastasiiaSergienko) [reply .github/workflows/ci-build.yml:4]: The lines formatting is gone...

Comment (jakobbraun) [reply .github/workflows/ci-build.yml]: I just tried it out, but it did not re-run the tests after a push on this PR...
image

Comment (jakobbraun) [reply .github/workflows/ci-build.yml]: image
(from https://docs.github.com/en/actions/reference/events-that-trigger-workflows)

Comment (jakobbraun) [reply .github/workflows/ci-build.yml]: So we could configure it, but I think simply using push if fine

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**